problems/arraysIntro.js

```diff
@@ -64,29 +92,50 @@ function arrayToString() {}
  * @param {Array} array2
  * @returns {string|Array}
  */
-function largerArray() {}
-
+function largerArray(array1, array2) {
+  if(array1.length > array2.length){
+    return array1;
+  } else if (array1.length === array2.length){
+    return "They are the same size"
+  }
+  else {
+    return array2;
+  }
```

Comment:
Make sure that you format you code using a extension like [prettier ](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)